sessionctx/variable/tidb_vars.go

```diff
@@ -761,6 +761,8 @@ const (
 	TiDBAuthSigningCert = "tidb_auth_signing_cert"
 	// TiDBAuthSigningKey indicates the path of the signing key to do token-based authentication.
 	TiDBAuthSigningKey = "tidb_auth_signing_key"
+	// TiDBGCAwareMemoryTrack indicates whether to turn-on GC-aware memory track.
+	TiDBGCAwareMemoryTrack = "tidb_gc_aware_memory_track"
```

Comment:
```suggestion
	TiDBEnableGCAwareMemoryTrack = "tidb_enable_gc_aware_memory_track"
```

I tend to introduce `enable` if this variable is used to control the feature which is common in other variables.

Reply:
Updated.